vignettes/advanced.Rmd

```diff
@@ -26,102 +29,290 @@ knitr::opts_chunk$set(
+```{r}
+draw_obj <- draws(
+    data = data,
+    data_ice = NULL, # if NULL, MAR is assumed for every missing data
+    vars = vars_tv,
+    method = method
+)
+
+impute_obj_tv <- impute(
+    draw_obj,
+    references = c("Control" = "Control", "Intervention" = "Intervention")
+)
+
+ana_obj_tv <- analyse(
+    impute_obj_tv,
+    vars = vars_an
+)
+
+pool(ana_obj_tv)
+```
 
 # Efficiently Changing Imputation Strategies
 
-The `draws()` function is by far the most computationally intensive function in `rbmi`. 
-In some settings, it may be important to explore the impact of a different 
-reference-based imputation strategy than the one chosen for the main analysis. 
-This change in the imputation strategy does not affect the imputation model but it does 
+The `draws()` function is by far the most computationally intensive function in `rbmi`.
+In some settings, it may be important to explore the impact of a different
+reference-based imputation strategy than the one chosen for the main analysis.
+This change in the imputation strategy does not affect the imputation model but it does
 affect the subsequent imputation step. 
 In order to allow changes in the imputation strategy without having to re-run the 
-`draws()` function, the function `impute()` has an additional argument `update_strategies`.  
+`draws()` function, the function `impute()` has an additional argument `update_strategies`.
 
 Please note though that this functionality comes with a some key limitations:
-From a theoretical point of view, such updates are only sensible if the base imputation model and the data to which it is fitted are also applicable to the revised imputation strategy. For example, as described in `help("draws", "rbmi")`, observed post-ICE data are included in the imputation model for the MAR strategy but excluded for non-MAR strategies. For this reason, the imputation strategy cannot be changed from MAR to a non-MAR strategy in the presence of observed data after the ICE. Likewise, a change from a non-MAR to MAR in the presence of observed data after the ICE triggers a warning because the imputation model was not fitted to all relevant data and may be inefficient. Similarly, it is not possible to change the first visit which is affected by an ICE via argument `update_strategies`. 
+From a theoretical point of view, such updates are only sensible if the base imputation model and the data to which it is fitted are also applicable to the revised imputation strategy.
+For example, as described in `help("draws", "rbmi")`, observed post-ICE data are included in the imputation model for the MAR strategy but excluded for non-MAR strategies.
+For this reason, the imputation strategy cannot be changed from MAR to a non-MAR strategy in the presence of observed data after the ICE.
+Likewise, a change from a non-MAR to MAR in the presence of observed data after the ICE triggers a warning because the imputation model was not fitted to all relevant data and may be inefficient.
```

Comment:
I still wonder if we should be more explicit than just "inefficient" as people could thing this means inefficient in terms of computing resources rather than not using all of the available data.

Reply:
We could just delete "..and may be inefficient"?

Reply:
Even if maybe it is clear enough that here inefficiency is not related to computing resources..